model/PermissionsService.php

```diff
@@ -228,14 +328,14 @@ private function validateResources(array $resultPermissions): void
         }
     }
 
-    /**
-     * @param array $addRemove
-     * @param string $resourceId
-     * @param bool $isRecursive
-     */
+    private function triggerEvents(
+        array $addRemove,
+        string $resourceId,
+        bool $isRecursive,
+        bool $applyToNestedResources
+    ): void {
+        $this->getLogger()->info(sprintf("Triggering events for %s", $resourceId));
```

Comment:
I am not sure this logs helps a lot. `Triggering events...` does not say much about which events. I would rather remove it

Reply:
I'm removing it (it was there to help me with debugging (resources & classes that were triggering events and in which order))